docs/reference/issues.md

```diff
@@ -0,0 +1,23 @@
+# Known Issues
+
+This page includes known issues, compatibility issues as well as possible workarounds.
+
+
+## Issue: Some ROS nodes have a noticable lag
+
+You should make sure that queue size is set to 1 and the buffer size is large enough to hold the input message.
+Even though we have set the appropriate default values for topics in order to avoid this issue, this also depends on your system configuration (e.g., size messages published in input topics).
+Be sure to check the discussion and explanatation of this behavior in #275.
```

Comment:
Is the reference working?
Otherwise we need to pass the full URL.
```suggestion
Be sure to check the discussion and explanatation of this behavior in [#275](https://github.com/opendr-eu/opendr/issues/275).
```